orcid-core/src/main/java/org/orcid/core/security/visibility/aop/OrcidApiAuthorizationSecurityAspect.java

```diff
@@ -366,5 +370,29 @@ private boolean allowAnonymousAccess(Authentication auth, AccessControl accessCo
         }
         return allowAnonymousAccess;
     } 
+    
+    /**
+     * Check if a member have a specific scope over a client
+     * 
+     * @param clientId
+     * @param userName
+     * @param scopes
+     * @return true if the member have access to any of the specified scope on the specified user
+     * */
+    @Deprecated
+    private boolean checkIfScopeIsAvailableForMember(String clientId, String userName, List<String> requiredScopes) {
```

Comment:
This function is tricky, it will return true if the client have any of the requiredScopes available for the specified user.
We should not use it anywhere, thats why it is private.